src/components/DatePicker/DatePicker.driver.ts

```diff
@@ -0,0 +1,159 @@
+     */
+    getCurrentMonthWithYear: async () =>
+      WSRCalendarDriver.getCurrentMonthWithYear(),
+    /**
+     * Get the the day name label of a specific date
+     * @param n - the day index
+     */
+    getNthWeekDayName: async (n: number) =>
+      WSRCalendarDriver.getNthWeekDayName(n),
+    /**
+     * Click on the Nth day in the current displayed month
+     * @param n - the day index
+     */
+    clickOnNthDay: async (n: number) => WSRCalendarDriver.clickOnNthDay(n),
+    /**
+     * Click on a given day
+     * @param date - a date to select
+     */
+    clickDay: async (date: Date) => WSRCalendarDriver.clickDay(date),
+    /**
+     * Check if the given day is active
+     * @param date - a date
+     */
+    isDayActive: async (date: Date) => WSRCalendarDriver.isDayActive(date),
+    /**
+     * Click on the year dropdown
+     */
+    clickOnYearDropdown: async () => WSRCalendarDriver.clickOnYearDropdown(),
+    /**
+     * Click on the month dropdown
+     */
+    clickOnMonthDropdown: async () => WSRCalendarDriver.clickOnMonthDropdown(),
+    /**
+     * Click on the Nth dropdown option in the year dropdown
+     * @param n - the index of the year in the list
+     */
+    clickOnNthYear: async (n: number) => WSRCalendarDriver.clickOnNthYear(n),
+    /**
+     * Click on the prev month button
+     */
+    clickOnPrevMonthButton: async () =>
+      WSRCalendarDriver.clickOnPrevMonthButton(),
+    /**
+     * Click on the next month button
+     */
+    clickOnNextMonthButton: async () =>
+      WSRCalendarDriver.clickOnNextMonthButton(),
+    /**
+     * Check whether the year dropdown exist
+     */
+    isYearDropdownExists: async () => WSRCalendarDriver.isYearDropdownExists(),
+    /**
+     * Check whether the year caption exist
+     */
+    isYearCaptionExists: async () => WSRCalendarDriver.isYearCaptionExists(),
+    /**
+     * Check whether the month dropdown exist
+     */
+    isMonthDropdownExists: async () =>
+      WSRCalendarDriver.isMonthDropdownExists(),
+    /**
+     * Check whether the month caption exist
+     */
+    isMonthCaptionExists: async () => WSRCalendarDriver.isMonthCaptionExists(),
+    /**
+     * Get the month caption text
+     */
+    getMonthCaption: async () => WSRCalendarDriver.getMonthCaption(),
+    /**
+     * Get the label of the selected month dropdown option
+     */
+    getMonthDropdownLabel: async () =>
+      WSRCalendarDriver.getMonthDropdownLabel(),
+    /**
+     * Get the label of the selected year dropdown option
+     */
+    getSelectedYear: async () => WSRCalendarDriver.getSelectedYear(),
+    /**
+     *  Returns the text of the focused day or `null` if there is no focused day
+     */
+    getFocusedDay: async () => WSRCalendarDriver.getFocusedDay(),
+    /**
+     * Press left arrow
+     */
+    pressLeftArrow: async () => WSRCalendarDriver.pressLeftArrow(),
```

Comment:
What's the difference between `clickOnPrevMonthButton` and this?
We may have talked about this already and I forgot 😄 
Is this because in RTL the sides have different meanings?

Maybe we don't need to expose both types of methods.

In any case, and not critical at all, but maybe we want `click` instead of `press` 🤔 ?
It's just that `press` usually refers to mobile.
Again, not critical

Reply:
The `clickOnPrevMonthButton` function is for clicking on the header's arrow. The header's arrows are responsible for changing months.

The `pressLeftArrow` function is simulating pressing on the left arrow in the keyboard.
It is used in WSR to test a11y.
There are many tests that check this functionality in WSR and I think we don't need to test it again. 

Actually, there are a lot of functions in the WSR driver that are exposed which I wasn't exposed in the TPA because I think that some of them are not relevant for the users. But maybe we should remove some more..

Reply:
Oh, I see, ok cool 👌 

I wonder if maybe we should change this to `pressLeftArrowKey` 🤔